lms/static/js/discovery/discovery_factory.js

```diff
@@ -72,7 +72,7 @@
                 });
 
                 dispatcher.listenTo(search, 'error', function() {
-                    form.showErrorMessage();
+                    form.showErrorMessage(search);
```

Comment:
I think `showErrorMessage` would be cleaner if this passed in `search.errorMessage`. Any reason that couldn’t be done?